Dockerfile

```diff
@@ -1,8 +1,7 @@
-FROM fedora:22
-MAINTAINER mscherer@redhat.com
+FROM fedora:23
+MAINTAINER jberkus@redhat.com
```

Comment:
I was thinking more of somethign like atomic-devel or similiar.  But it isn't a big deal either way.